### PR TITLE
Release 2.1

### PR DESCRIPTION
**Pull request by rayzhou2017**

### CODE REVIEW

File: src/templates/markdown.js [hunk 1]
```diff
@@ -225,6 +225,16 @@ class MarkdownTemplate extends React.Component {
         )
       : postNode.html
```
Comment (FeynmanZhou) [inline src/templates/markdown.js:227]: This PR contains the commit from @leoendless , will these `.js` files influence the current branch? @leoendless

Comment (leoendless) [reply src/templates/markdown.js:227]: Yes, this pr need to rebase the latest release-2.1 branch.

Comment (FeynmanZhou) [reply src/templates/markdown.js:227]: @rayzhou2017 Could you please rebase it?

Comment (rayzhou2017) [reply src/templates/markdown.js:235]: rebased

Comment (leoendless) [reply src/templates/markdown.js:235]: The PR should rebase branch ``release-2.1``. No need to change the javascript files.

File: content/en/release/release-v210.md [hunk 2]
```diff
@@ -47,20 +47,22 @@ Fix the StorageClass list display problem
 
 ### Features
 
-- Support for collecting the file logs on the disk,  it is used for the Pod which preserve the logs as the file on the disk
+- Support for collecting the file logs on the disk. It is used for the Pod which preserves the logs as the file on the disk
 - Support integrating with external ElasticSearch 7.x
-- Ability to search logs including Chinese words
+- Ability to search logs containinh Chinese words
```
Comment (FeynmanZhou) [inline content/en/release/release-v210.md:52]: containinh => containing

**Pull request merged**